driver/registry_default.go

```diff
@@ -216,20 +227,18 @@ func (m *RegistryDefault) HealthHandler(_ context.Context) *healthx.Handler {
 				"database": func(_ *http.Request) error {
 					return m.Ping()
 				},
-				// fandom-start: disable migrations check which fails on non-local DBs
-				//"migrations": func(r *http.Request) error {
-				//	status, err := m.Persister().MigrationStatus(r.Context())
-				//	if err != nil {
-				//		return err
-				//	}
-				//
-				//	if status.HasPending() {
-				//		return errors.Errorf("migrations have not yet been fully applied")
-				//	}
-				//
-				//	return nil
-				//},
-				// fandom-end
+				"migrations": func(r *http.Request) error {
+					status, err := m.Persister().MigrationStatus(r.Context())
+					if err != nil {
+						return err
+					}
+
+					if status.HasPending() {
+						return errors.Errorf("migrations have not yet been fully applied")
+					}
+
+					return nil
+				},
```

Comment:
shouldn't this still be disabled?

Reply:
I believe we should be OK now after switching to ProxySQL. We can re-disable this when we notice issues (it should be visible straight after the first deploy).